src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeregningBarnetilsynUtil.kt

```diff
@@ -43,43 +45,45 @@ object BeregningBarnetilsynUtil {
                                                    utgifter = utgiftsperiode.utgifter,
                                                    kontantstøttebeløp = kontantstøtteBeløp,
                                                    tilleggsstønadsbeløp = tilleggsstønadBeløp,
-                                                   antallBarn = antallBarnIPeriode))
+                                                   antallBarn = barn.size,
+                                                   barn = barn))
     }
 
-    data class BeregnedeBeløp(val utbetaltBeløp: BigDecimal, val beløpFørSatsjustering: BigDecimal)
+        data class BeregnedeBeløp(val utbetaltBeløp: BigDecimal, val beløpFørSatsjustering: BigDecimal)
```

Comment:
Her har det skjedd noe med formatteringen? 👀  (i hela filen)

Reply:
Jeg oppdaterte Inntellij litt for tidlig, og nå er alle settingsene mine endret 😨  Så må få lagt inn den formateringsfilen på nytt

Reply:
Troor jeg fiksa det nå :D